src/main/java/org/symphonyoss/symphony/messageml/elements/Select.java

```diff
@@ -75,21 +76,14 @@ protected void buildAttribute(org.w3c.dom.Node item) throws InvalidInputExceptio
     }
   }
 
-  private void assertAtLeastOneOptionChild() throws InvalidInputException {
-    boolean hasOptionElementAsChild = this.getChildren()
-        .stream()
-        .anyMatch(child -> child.getClass().equals(Option.class));
-
-    if (!hasOptionElementAsChild) {
-      throw new InvalidInputException(String.format("The \"%s\" element must have at least one \"%s\" as its child.",
-          MESSAGEML_TAG, Option.MESSAGEML_TAG));
-    }
-  }
+  private void assertOnlyOneOptionSelected() throws InvalidInputException {
```

Comment:
In contrast to my remarks about other validations, this one looks like a good example of a single-use only assert which actually belongs to this class.